test/e2e/linuxbridge_test.go

```diff
@@ -0,0 +1,106 @@
+package e2e
+
+import (
+	"context"
+	"io/ioutil"
+	"testing"
+	"time"
+
+	yaml "github.com/ghodss/yaml"
+
+	apis "github.com/nmstate/kubernetes-nmstate/pkg/apis"
+	nmstatev1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1"
+
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	"github.com/operator-framework/operator-sdk/pkg/test/e2eutil"
+
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var (
+	retryInterval        = time.Second * 5
+	timeout              = time.Second * 60
+	cleanupRetryInterval = time.Second * 1
+	cleanupTimeout       = time.Second * 5
+	createBridgeCr       = "deploy/crds/create-br1-linux-bridge.yaml"
+	deleteBridgeCr       = "deploy/crds/delete-br1-linux-bridge.yaml"
+	firstNodeName        = "node01" // TODO: Get it from cluster
+	bridgeName           = "br1"
+)
+
+func TestLinuxBridge(t *testing.T) {
+
+	nodeNetworkStateList := &nmstatev1.NodeNetworkStateList{}
+	err := framework.AddToFrameworkScheme(apis.AddToScheme, nodeNetworkStateList)
+	if err != nil {
+		t.Fatalf("failed to add custom resource scheme to framework: %v", err)
+	}
+
+	ctx := framework.NewTestCtx(t)
+	defer ctx.Cleanup()
+
+	err = ctx.InitializeClusterResources(&framework.CleanupOptions{TestContext: ctx, Timeout: cleanupTimeout, RetryInterval: cleanupRetryInterval})
+	if err != nil {
+		t.Fatalf("failed to initialize cluster resources: %v", err)
+	}
+
+	// get namespace
+	namespace, err := ctx.GetNamespace()
+	if err != nil {
+		t.Fatal(err)
+	}
+	// get global framework variables
```

Comment:
please keep and empty line after checking for an error and before assigning a new values. feel free to ignore it, i should stop nit picking and write a script that would fail CI if there is something i don't like

Reply:
Let's re-review after ginkgo refactoring, since this was a WIP.